src/oaklib/cli.py

```diff
@@ -3489,6 +3508,14 @@ def lexmatch(output, recreate, rules_file, lexical_index_file, add_labels, terms
         ruleset = load_mapping_rules(rules_file)
     else:
         ruleset = None
+
+    if exclude_terms:
+        terms_to_remove = []
+        if len(exclude_terms) == 1 and Path(exclude_terms[0]).exists():
+            with open(exclude_terms[0]) as f:
+                terms_to_remove = f.read().splitlines()
+        else:
+            terms_to_remove = list(exclude_terms)
```

Comment:
it looks like nothing is done with this

Reply:
Yeah I was going to start with `lexmatch` but realized `annotate` has an immediate need for implementation. I've commented the code out for now.